package.json

```diff
@@ -24,7 +26,10 @@
     "eslint": "^4.11.0"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-router": "^4.0.0 || ^5.0.0"
+    "react": "^16.8.0 || ^17.0.0",
+    "react-router-dom": "^4.0.0 || ^5.0.0"
```

Comment:
This I believe this change makes this PR a breaking change? Do you think it's worth the major bump?

Reply:
Should we only be importing the dom package in our [`client-template`](https://github.com/LaunchPadLab/client-template/blob/6478cc0b58774a963a6c4c2a42a107ba2aec276e/package.json#L66-L67)?